packaging/suse/portus.spec.in

```diff
@@ -52,16 +62,35 @@ Obsoletes:      Portus = 20151120162040
 BuildRequires:  nodejs6
 BuildRequires:  yarn
 
+# If we require yarn-packaging, this will auto create the provides
+# for the js libs
+BuildRequires:  yarn-packaging
```

Comment:
Maybe this can be merged with the above code on JS, so it's left like this:

```
# Javascript engine to build assets. Note that yarn-packaging will automatically create the provides
# for the JS libs
BuildRequires: nodejs6
BuildRequires: yarn
BuildRequires: yarn-packaging
```

Reply:
done.